rllib/evaluation/collectors/agent_collector.py

```diff
@@ -359,11 +365,11 @@ def build_for_training(
             SampleBatch: The built SampleBatch for this agent, ready to go into
             postprocessing.
         """
-
         batch_data = {}
         np_data = {}
         for view_col, view_req in view_requirements.items():
             # Create the batch of data from the different buffers.
+            # "state_in_0"
```

Comment:
Left this here by accident? I don't understand what it's trying to tell me

Reply:
yep